[2024-10-02T18:55:48.911+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-10-02T18:55:48.939+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sparking_flow.python_job scheduled__2024-10-01T00:00:00+00:00 [queued]>
[2024-10-02T18:55:48.951+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sparking_flow.python_job scheduled__2024-10-01T00:00:00+00:00 [queued]>
[2024-10-02T18:55:48.951+0000] {taskinstance.py:2865} INFO - Starting attempt 2 of 2
[2024-10-02T18:55:48.972+0000] {taskinstance.py:2888} INFO - Executing <Task(SparkSubmitOperator): python_job> on 2024-10-01 00:00:00+00:00
[2024-10-02T18:55:48.981+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'sparking_flow', 'python_job', 'scheduled__2024-10-01T00:00:00+00:00', '--job-id', '13455', '--raw', '--subdir', 'DAGS_FOLDER/materi/test.py', '--cfg-path', '/tmp/tmpss385x6t']
[2024-10-02T18:55:48.983+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=78270) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-10-02T18:55:48.984+0000] {standard_task_runner.py:105} INFO - Job 13455: Subtask python_job
[2024-10-02T18:55:48.984+0000] {standard_task_runner.py:72} INFO - Started process 78271 to run task
[2024-10-02T18:55:49.060+0000] {task_command.py:467} INFO - Running <TaskInstance: sparking_flow.python_job scheduled__2024-10-01T00:00:00+00:00 [running]> on host 19cd1516b3ca
[2024-10-02T18:55:49.201+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='Yusuf Ganiyu' AIRFLOW_CTX_DAG_ID='sparking_flow' AIRFLOW_CTX_TASK_ID='python_job' AIRFLOW_CTX_EXECUTION_DATE='2024-10-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-10-01T00:00:00+00:00'
[2024-10-02T18:55:49.202+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-10-02T18:55:49.244+0000] {base.py:84} INFO - Retrieving connection 'spark-conn'
[2024-10-02T18:55:49.245+0000] {spark_submit.py:335} INFO - Spark-Submit cmd: spark-submit --master spark://spark-master:7077 --name arrow-spark jobs/wordcountjob.py
[2024-10-02T18:55:49.312+0000] {spark_submit.py:488} INFO - /home/***/.local/lib/python3.12/site-packages/pyspark/bin/load-spark-env.sh: line 68: ps: command not found
[2024-10-02T18:55:56.114+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:56 INFO SparkContext: Running Spark version 3.5.3
[2024-10-02T18:55:56.120+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:56 INFO SparkContext: OS info Linux, 5.15.153.1-microsoft-standard-WSL2, amd64
[2024-10-02T18:55:56.122+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:56 INFO SparkContext: Java version 17.0.12
[2024-10-02T18:55:56.231+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:56 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-10-02T18:55:56.563+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:56 INFO ResourceUtils: ==============================================================
[2024-10-02T18:55:56.564+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:56 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-10-02T18:55:56.565+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:56 INFO ResourceUtils: ==============================================================
[2024-10-02T18:55:56.566+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:56 INFO SparkContext: Submitted application: PythonWordCount
[2024-10-02T18:55:56.615+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:56 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-10-02T18:55:56.633+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:56 INFO ResourceProfile: Limiting resource is cpu
[2024-10-02T18:55:56.639+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:56 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-10-02T18:55:56.803+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:56 INFO SecurityManager: Changing view acls to: ***
[2024-10-02T18:55:56.805+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:56 INFO SecurityManager: Changing modify acls to: ***
[2024-10-02T18:55:56.808+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:56 INFO SecurityManager: Changing view acls groups to:
[2024-10-02T18:55:56.811+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:56 INFO SecurityManager: Changing modify acls groups to:
[2024-10-02T18:55:56.815+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:56 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: ***; groups with view permissions: EMPTY; users with modify permissions: ***; groups with modify permissions: EMPTY
[2024-10-02T18:55:57.647+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:57 INFO Utils: Successfully started service 'sparkDriver' on port 35347.
[2024-10-02T18:55:57.894+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:57 INFO SparkEnv: Registering MapOutputTracker
[2024-10-02T18:55:58.011+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:58 INFO SparkEnv: Registering BlockManagerMaster
[2024-10-02T18:55:58.060+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:58 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-10-02T18:55:58.062+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:58 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-10-02T18:55:58.072+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:58 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-10-02T18:55:58.130+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:58 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-db463b4c-a45c-41c7-ba3a-36ebe5675b62
[2024-10-02T18:55:58.171+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:58 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-10-02T18:55:58.221+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:58 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-10-02T18:55:58.457+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:58 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2024-10-02T18:55:58.617+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:58 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2024-10-02T18:55:58.913+0000] {spark_submit.py:488} INFO - 24/10/02 18:55:58 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark-master:7077...
[2024-10-02T18:56:06.965+0000] {spark_submit.py:488} INFO - 24/10/02 18:56:06 WARN TransportClientFactory: DNS resolution failed for spark-master/<unresolved>:7077 took 8019 ms
[2024-10-02T18:56:07.087+0000] {spark_submit.py:488} INFO - 24/10/02 18:56:07 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark-master:7077
[2024-10-02T18:56:07.088+0000] {spark_submit.py:488} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2024-10-02T18:56:07.088+0000] {spark_submit.py:488} INFO - at org.apache.spark.util.SparkThreadUtils$.awaitResult(SparkThreadUtils.scala:56)
[2024-10-02T18:56:07.089+0000] {spark_submit.py:488} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:310)
[2024-10-02T18:56:07.090+0000] {spark_submit.py:488} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2024-10-02T18:56:07.090+0000] {spark_submit.py:488} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:102)
[2024-10-02T18:56:07.091+0000] {spark_submit.py:488} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:110)
[2024-10-02T18:56:07.091+0000] {spark_submit.py:488} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:108)
[2024-10-02T18:56:07.091+0000] {spark_submit.py:488} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
[2024-10-02T18:56:07.092+0000] {spark_submit.py:488} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2024-10-02T18:56:07.092+0000] {spark_submit.py:488} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2024-10-02T18:56:07.092+0000] {spark_submit.py:488} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2024-10-02T18:56:07.093+0000] {spark_submit.py:488} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2024-10-02T18:56:07.093+0000] {spark_submit.py:488} INFO - Caused by: java.io.IOException: Failed to connect to spark-master/<unresolved>:7077
[2024-10-02T18:56:07.093+0000] {spark_submit.py:488} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:294)
[2024-10-02T18:56:07.094+0000] {spark_submit.py:488} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:214)
[2024-10-02T18:56:07.094+0000] {spark_submit.py:488} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:226)
[2024-10-02T18:56:07.095+0000] {spark_submit.py:488} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2024-10-02T18:56:07.095+0000] {spark_submit.py:488} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2024-10-02T18:56:07.095+0000] {spark_submit.py:488} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2024-10-02T18:56:07.096+0000] {spark_submit.py:488} INFO - ... 4 more
[2024-10-02T18:56:07.096+0000] {spark_submit.py:488} INFO - Caused by: java.net.UnknownHostException: spark-master
[2024-10-02T18:56:07.096+0000] {spark_submit.py:488} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:801)
[2024-10-02T18:56:07.097+0000] {spark_submit.py:488} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1533)
[2024-10-02T18:56:07.098+0000] {spark_submit.py:488} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1385)
[2024-10-02T18:56:07.098+0000] {spark_submit.py:488} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1306)
[2024-10-02T18:56:07.098+0000] {spark_submit.py:488} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1256)
[2024-10-02T18:56:07.100+0000] {spark_submit.py:488} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2024-10-02T18:56:07.101+0000] {spark_submit.py:488} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2024-10-02T18:56:07.101+0000] {spark_submit.py:488} INFO - at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
[2024-10-02T18:56:07.102+0000] {spark_submit.py:488} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2024-10-02T18:56:07.102+0000] {spark_submit.py:488} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2024-10-02T18:56:07.103+0000] {spark_submit.py:488} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2024-10-02T18:56:07.103+0000] {spark_submit.py:488} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2024-10-02T18:56:07.104+0000] {spark_submit.py:488} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2024-10-02T18:56:07.105+0000] {spark_submit.py:488} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2024-10-02T18:56:07.105+0000] {spark_submit.py:488} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2024-10-02T18:56:07.106+0000] {spark_submit.py:488} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:220)
[2024-10-02T18:56:07.106+0000] {spark_submit.py:488} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2024-10-02T18:56:07.107+0000] {spark_submit.py:488} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:189)
[2024-10-02T18:56:07.107+0000] {spark_submit.py:488} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:175)
[2024-10-02T18:56:07.108+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:590)
[2024-10-02T18:56:07.108+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:557)
[2024-10-02T18:56:07.109+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:492)
[2024-10-02T18:56:07.110+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:636)
[2024-10-02T18:56:07.110+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:625)
[2024-10-02T18:56:07.111+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:105)
[2024-10-02T18:56:07.111+0000] {spark_submit.py:488} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2024-10-02T18:56:07.112+0000] {spark_submit.py:488} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:990)
[2024-10-02T18:56:07.113+0000] {spark_submit.py:488} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2024-10-02T18:56:07.113+0000] {spark_submit.py:488} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2024-10-02T18:56:07.114+0000] {spark_submit.py:488} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2024-10-02T18:56:07.114+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174)
[2024-10-02T18:56:07.115+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167)
[2024-10-02T18:56:07.117+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470)
[2024-10-02T18:56:07.118+0000] {spark_submit.py:488} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:569)
[2024-10-02T18:56:07.119+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
[2024-10-02T18:56:07.119+0000] {spark_submit.py:488} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2024-10-02T18:56:07.120+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2024-10-02T18:56:07.120+0000] {spark_submit.py:488} INFO - ... 1 more
[2024-10-02T18:56:18.915+0000] {spark_submit.py:488} INFO - 24/10/02 18:56:18 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark-master:7077...
[2024-10-02T18:56:26.924+0000] {spark_submit.py:488} INFO - 24/10/02 18:56:26 WARN TransportClientFactory: DNS resolution failed for spark-master/<unresolved>:7077 took 8005 ms
[2024-10-02T18:56:26.931+0000] {spark_submit.py:488} INFO - 24/10/02 18:56:26 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark-master:7077
[2024-10-02T18:56:26.931+0000] {spark_submit.py:488} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2024-10-02T18:56:26.932+0000] {spark_submit.py:488} INFO - at org.apache.spark.util.SparkThreadUtils$.awaitResult(SparkThreadUtils.scala:56)
[2024-10-02T18:56:26.933+0000] {spark_submit.py:488} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:310)
[2024-10-02T18:56:26.934+0000] {spark_submit.py:488} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2024-10-02T18:56:26.935+0000] {spark_submit.py:488} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:102)
[2024-10-02T18:56:26.935+0000] {spark_submit.py:488} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:110)
[2024-10-02T18:56:26.936+0000] {spark_submit.py:488} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:108)
[2024-10-02T18:56:26.936+0000] {spark_submit.py:488} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
[2024-10-02T18:56:26.936+0000] {spark_submit.py:488} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2024-10-02T18:56:26.937+0000] {spark_submit.py:488} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2024-10-02T18:56:26.937+0000] {spark_submit.py:488} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2024-10-02T18:56:26.937+0000] {spark_submit.py:488} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2024-10-02T18:56:26.937+0000] {spark_submit.py:488} INFO - Caused by: java.io.IOException: Failed to connect to spark-master/<unresolved>:7077
[2024-10-02T18:56:26.937+0000] {spark_submit.py:488} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:294)
[2024-10-02T18:56:26.938+0000] {spark_submit.py:488} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:214)
[2024-10-02T18:56:26.938+0000] {spark_submit.py:488} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:226)
[2024-10-02T18:56:26.938+0000] {spark_submit.py:488} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2024-10-02T18:56:26.939+0000] {spark_submit.py:488} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2024-10-02T18:56:26.939+0000] {spark_submit.py:488} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2024-10-02T18:56:26.940+0000] {spark_submit.py:488} INFO - ... 4 more
[2024-10-02T18:56:26.940+0000] {spark_submit.py:488} INFO - Caused by: java.net.UnknownHostException: spark-master
[2024-10-02T18:56:26.941+0000] {spark_submit.py:488} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:801)
[2024-10-02T18:56:26.941+0000] {spark_submit.py:488} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1533)
[2024-10-02T18:56:26.941+0000] {spark_submit.py:488} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1385)
[2024-10-02T18:56:26.942+0000] {spark_submit.py:488} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1306)
[2024-10-02T18:56:26.942+0000] {spark_submit.py:488} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1256)
[2024-10-02T18:56:26.942+0000] {spark_submit.py:488} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2024-10-02T18:56:26.943+0000] {spark_submit.py:488} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2024-10-02T18:56:26.943+0000] {spark_submit.py:488} INFO - at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
[2024-10-02T18:56:26.943+0000] {spark_submit.py:488} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2024-10-02T18:56:26.944+0000] {spark_submit.py:488} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2024-10-02T18:56:26.944+0000] {spark_submit.py:488} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2024-10-02T18:56:26.944+0000] {spark_submit.py:488} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2024-10-02T18:56:26.944+0000] {spark_submit.py:488} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2024-10-02T18:56:26.945+0000] {spark_submit.py:488} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2024-10-02T18:56:26.945+0000] {spark_submit.py:488} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2024-10-02T18:56:26.946+0000] {spark_submit.py:488} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:220)
[2024-10-02T18:56:26.946+0000] {spark_submit.py:488} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2024-10-02T18:56:26.946+0000] {spark_submit.py:488} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:189)
[2024-10-02T18:56:26.947+0000] {spark_submit.py:488} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:175)
[2024-10-02T18:56:26.948+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:590)
[2024-10-02T18:56:26.948+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:557)
[2024-10-02T18:56:26.949+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:492)
[2024-10-02T18:56:26.950+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:636)
[2024-10-02T18:56:26.950+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:625)
[2024-10-02T18:56:26.950+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:105)
[2024-10-02T18:56:26.951+0000] {spark_submit.py:488} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2024-10-02T18:56:26.952+0000] {spark_submit.py:488} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:990)
[2024-10-02T18:56:26.952+0000] {spark_submit.py:488} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2024-10-02T18:56:26.953+0000] {spark_submit.py:488} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2024-10-02T18:56:26.953+0000] {spark_submit.py:488} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2024-10-02T18:56:26.953+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174)
[2024-10-02T18:56:26.954+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167)
[2024-10-02T18:56:26.954+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470)
[2024-10-02T18:56:26.954+0000] {spark_submit.py:488} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:569)
[2024-10-02T18:56:26.954+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
[2024-10-02T18:56:26.954+0000] {spark_submit.py:488} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2024-10-02T18:56:26.955+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2024-10-02T18:56:26.955+0000] {spark_submit.py:488} INFO - ... 1 more
[2024-10-02T18:56:38.916+0000] {spark_submit.py:488} INFO - 24/10/02 18:56:38 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark-master:7077...
[2024-10-02T18:56:46.921+0000] {spark_submit.py:488} INFO - 24/10/02 18:56:46 WARN TransportClientFactory: DNS resolution failed for spark-master/<unresolved>:7077 took 8003 ms
[2024-10-02T18:56:46.930+0000] {spark_submit.py:488} INFO - 24/10/02 18:56:46 WARN StandaloneAppClient$ClientEndpoint: Failed to connect to master spark-master:7077
[2024-10-02T18:56:46.931+0000] {spark_submit.py:488} INFO - org.apache.spark.SparkException: Exception thrown in awaitResult:
[2024-10-02T18:56:46.932+0000] {spark_submit.py:488} INFO - at org.apache.spark.util.SparkThreadUtils$.awaitResult(SparkThreadUtils.scala:56)
[2024-10-02T18:56:46.932+0000] {spark_submit.py:488} INFO - at org.apache.spark.util.ThreadUtils$.awaitResult(ThreadUtils.scala:310)
[2024-10-02T18:56:46.933+0000] {spark_submit.py:488} INFO - at org.apache.spark.rpc.RpcTimeout.awaitResult(RpcTimeout.scala:75)
[2024-10-02T18:56:46.934+0000] {spark_submit.py:488} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRefByURI(RpcEnv.scala:102)
[2024-10-02T18:56:46.935+0000] {spark_submit.py:488} INFO - at org.apache.spark.rpc.RpcEnv.setupEndpointRef(RpcEnv.scala:110)
[2024-10-02T18:56:46.936+0000] {spark_submit.py:488} INFO - at org.apache.spark.deploy.client.StandaloneAppClient$ClientEndpoint$$anon$1.run(StandaloneAppClient.scala:108)
[2024-10-02T18:56:46.936+0000] {spark_submit.py:488} INFO - at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
[2024-10-02T18:56:46.937+0000] {spark_submit.py:488} INFO - at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
[2024-10-02T18:56:46.938+0000] {spark_submit.py:488} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
[2024-10-02T18:56:46.938+0000] {spark_submit.py:488} INFO - at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
[2024-10-02T18:56:46.939+0000] {spark_submit.py:488} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2024-10-02T18:56:46.940+0000] {spark_submit.py:488} INFO - Caused by: java.io.IOException: Failed to connect to spark-master/<unresolved>:7077
[2024-10-02T18:56:46.946+0000] {spark_submit.py:488} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:294)
[2024-10-02T18:56:46.947+0000] {spark_submit.py:488} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:214)
[2024-10-02T18:56:46.950+0000] {spark_submit.py:488} INFO - at org.apache.spark.network.client.TransportClientFactory.createClient(TransportClientFactory.java:226)
[2024-10-02T18:56:46.952+0000] {spark_submit.py:488} INFO - at org.apache.spark.rpc.netty.NettyRpcEnv.createClient(NettyRpcEnv.scala:204)
[2024-10-02T18:56:46.953+0000] {spark_submit.py:488} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:202)
[2024-10-02T18:56:46.954+0000] {spark_submit.py:488} INFO - at org.apache.spark.rpc.netty.Outbox$$anon$1.call(Outbox.scala:198)
[2024-10-02T18:56:46.954+0000] {spark_submit.py:488} INFO - ... 4 more
[2024-10-02T18:56:46.955+0000] {spark_submit.py:488} INFO - Caused by: java.net.UnknownHostException: spark-master
[2024-10-02T18:56:46.956+0000] {spark_submit.py:488} INFO - at java.base/java.net.InetAddress$CachedAddresses.get(InetAddress.java:801)
[2024-10-02T18:56:46.957+0000] {spark_submit.py:488} INFO - at java.base/java.net.InetAddress.getAllByName0(InetAddress.java:1533)
[2024-10-02T18:56:46.963+0000] {spark_submit.py:488} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1385)
[2024-10-02T18:56:46.964+0000] {spark_submit.py:488} INFO - at java.base/java.net.InetAddress.getAllByName(InetAddress.java:1306)
[2024-10-02T18:56:46.965+0000] {spark_submit.py:488} INFO - at java.base/java.net.InetAddress.getByName(InetAddress.java:1256)
[2024-10-02T18:56:46.966+0000] {spark_submit.py:488} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:156)
[2024-10-02T18:56:46.967+0000] {spark_submit.py:488} INFO - at io.netty.util.internal.SocketUtils$8.run(SocketUtils.java:153)
[2024-10-02T18:56:46.969+0000] {spark_submit.py:488} INFO - at java.base/java.security.AccessController.doPrivileged(AccessController.java:569)
[2024-10-02T18:56:46.970+0000] {spark_submit.py:488} INFO - at io.netty.util.internal.SocketUtils.addressByName(SocketUtils.java:153)
[2024-10-02T18:56:46.970+0000] {spark_submit.py:488} INFO - at io.netty.resolver.DefaultNameResolver.doResolve(DefaultNameResolver.java:41)
[2024-10-02T18:56:46.971+0000] {spark_submit.py:488} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:61)
[2024-10-02T18:56:46.972+0000] {spark_submit.py:488} INFO - at io.netty.resolver.SimpleNameResolver.resolve(SimpleNameResolver.java:53)
[2024-10-02T18:56:46.973+0000] {spark_submit.py:488} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:55)
[2024-10-02T18:56:46.973+0000] {spark_submit.py:488} INFO - at io.netty.resolver.InetSocketAddressResolver.doResolve(InetSocketAddressResolver.java:31)
[2024-10-02T18:56:46.974+0000] {spark_submit.py:488} INFO - at io.netty.resolver.AbstractAddressResolver.resolve(AbstractAddressResolver.java:106)
[2024-10-02T18:56:46.976+0000] {spark_submit.py:488} INFO - at io.netty.bootstrap.Bootstrap.doResolveAndConnect0(Bootstrap.java:220)
[2024-10-02T18:56:46.976+0000] {spark_submit.py:488} INFO - at io.netty.bootstrap.Bootstrap.access$000(Bootstrap.java:46)
[2024-10-02T18:56:46.978+0000] {spark_submit.py:488} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:189)
[2024-10-02T18:56:46.981+0000] {spark_submit.py:488} INFO - at io.netty.bootstrap.Bootstrap$1.operationComplete(Bootstrap.java:175)
[2024-10-02T18:56:46.982+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListener0(DefaultPromise.java:590)
[2024-10-02T18:56:46.983+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListenersNow(DefaultPromise.java:557)
[2024-10-02T18:56:46.983+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.DefaultPromise.notifyListeners(DefaultPromise.java:492)
[2024-10-02T18:56:46.984+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.DefaultPromise.setValue0(DefaultPromise.java:636)
[2024-10-02T18:56:46.985+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.DefaultPromise.setSuccess0(DefaultPromise.java:625)
[2024-10-02T18:56:46.985+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.DefaultPromise.trySuccess(DefaultPromise.java:105)
[2024-10-02T18:56:46.986+0000] {spark_submit.py:488} INFO - at io.netty.channel.DefaultChannelPromise.trySuccess(DefaultChannelPromise.java:84)
[2024-10-02T18:56:46.987+0000] {spark_submit.py:488} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.safeSetSuccess(AbstractChannel.java:990)
[2024-10-02T18:56:46.987+0000] {spark_submit.py:488} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.register0(AbstractChannel.java:516)
[2024-10-02T18:56:46.988+0000] {spark_submit.py:488} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe.access$200(AbstractChannel.java:429)
[2024-10-02T18:56:46.992+0000] {spark_submit.py:488} INFO - at io.netty.channel.AbstractChannel$AbstractUnsafe$1.run(AbstractChannel.java:486)
[2024-10-02T18:56:46.999+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174)
[2024-10-02T18:56:46.999+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167)
[2024-10-02T18:56:47.000+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470)
[2024-10-02T18:56:47.001+0000] {spark_submit.py:488} INFO - at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:569)
[2024-10-02T18:56:47.002+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
[2024-10-02T18:56:47.006+0000] {spark_submit.py:488} INFO - at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
[2024-10-02T18:56:47.008+0000] {spark_submit.py:488} INFO - at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
[2024-10-02T18:56:47.010+0000] {spark_submit.py:488} INFO - ... 1 more
[2024-10-02T18:56:58.921+0000] {spark_submit.py:488} INFO - 24/10/02 18:56:58 WARN StandaloneSchedulerBackend: Application ID is not initialized yet.
[2024-10-02T18:56:58.925+0000] {spark_submit.py:488} INFO - 24/10/02 18:56:58 ERROR StandaloneSchedulerBackend: Application has been killed. Reason: All masters are unresponsive! Giving up.
[2024-10-02T18:56:58.987+0000] {spark_submit.py:488} INFO - 24/10/02 18:56:58 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 42521.
[2024-10-02T18:56:58.996+0000] {spark_submit.py:488} INFO - 24/10/02 18:56:58 INFO NettyBlockTransferService: Server created on 19cd1516b3ca:42521
[2024-10-02T18:56:58.998+0000] {spark_submit.py:488} INFO - 24/10/02 18:56:58 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2024-10-02T18:56:59.063+0000] {spark_submit.py:488} INFO - 24/10/02 18:56:59 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, 19cd1516b3ca, 42521, None)
[2024-10-02T18:56:59.074+0000] {spark_submit.py:488} INFO - 24/10/02 18:56:59 INFO BlockManagerMasterEndpoint: Registering block manager 19cd1516b3ca:42521 with 434.4 MiB RAM, BlockManagerId(driver, 19cd1516b3ca, 42521, None)
[2024-10-02T18:56:59.087+0000] {spark_submit.py:488} INFO - 24/10/02 18:56:59 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, 19cd1516b3ca, 42521, None)
[2024-10-02T18:56:59.092+0000] {spark_submit.py:488} INFO - 24/10/02 18:56:59 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, 19cd1516b3ca, 42521, None)
[2024-10-02T18:57:00.469+0000] {spark_submit.py:488} INFO - 24/10/02 18:57:00 INFO StandaloneSchedulerBackend: SchedulerBackend is ready for scheduling beginning after reached minRegisteredResourcesRatio: 0.0
[2024-10-02T18:57:01.488+0000] {spark_submit.py:488} INFO - 24/10/02 18:57:01 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2024-10-02T18:57:01.522+0000] {spark_submit.py:488} INFO - 24/10/02 18:57:01 INFO SparkUI: Stopped Spark web UI at http://19cd1516b3ca:4040
[2024-10-02T18:57:01.540+0000] {spark_submit.py:488} INFO - 24/10/02 18:57:01 INFO StandaloneSchedulerBackend: Shutting down all executors
[2024-10-02T18:57:01.551+0000] {spark_submit.py:488} INFO - 24/10/02 18:57:01 INFO StandaloneSchedulerBackend$StandaloneDriverEndpoint: Asking each executor to shut down
[2024-10-02T18:57:01.566+0000] {spark_submit.py:488} INFO - 24/10/02 18:57:01 WARN StandaloneAppClient$ClientEndpoint: Drop UnregisterApplication(null) because has not yet connected to master
[2024-10-02T18:57:01.592+0000] {spark_submit.py:488} INFO - 24/10/02 18:57:01 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2024-10-02T18:57:01.632+0000] {spark_submit.py:488} INFO - 24/10/02 18:57:01 INFO MemoryStore: MemoryStore cleared
[2024-10-02T18:57:01.633+0000] {spark_submit.py:488} INFO - 24/10/02 18:57:01 INFO BlockManager: BlockManager stopped
[2024-10-02T18:57:01.662+0000] {spark_submit.py:488} INFO - 24/10/02 18:57:01 INFO BlockManagerMaster: BlockManagerMaster stopped
[2024-10-02T18:57:01.676+0000] {spark_submit.py:488} INFO - 24/10/02 18:57:01 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2024-10-02T18:57:01.781+0000] {spark_submit.py:488} INFO - 24/10/02 18:57:01 INFO SparkContext: Successfully stopped SparkContext
[2024-10-02T18:57:02.040+0000] {spark_submit.py:488} INFO - Traceback (most recent call last):
[2024-10-02T18:57:02.041+0000] {spark_submit.py:488} INFO - File "/opt/***/jobs/wordcountjob.py", line 11, in <module>
[2024-10-02T18:57:02.042+0000] {spark_submit.py:488} INFO - for wc in wordCounts.collect():
[2024-10-02T18:57:02.042+0000] {spark_submit.py:488} INFO - ^^^^^^^^^^^^^^^^^^^^
[2024-10-02T18:57:02.043+0000] {spark_submit.py:488} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/pyspark.zip/pyspark/rdd.py", line 1833, in collect
[2024-10-02T18:57:02.043+0000] {spark_submit.py:488} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/java_gateway.py", line 1322, in __call__
[2024-10-02T18:57:02.044+0000] {spark_submit.py:488} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/pyspark.zip/pyspark/errors/exceptions/captured.py", line 179, in deco
[2024-10-02T18:57:02.045+0000] {spark_submit.py:488} INFO - File "/home/***/.local/lib/python3.12/site-packages/pyspark/python/lib/py4j-0.10.9.7-src.zip/py4j/protocol.py", line 326, in get_return_value
[2024-10-02T18:57:02.045+0000] {spark_submit.py:488} INFO - py4j.protocol.Py4JJavaError: An error occurred while calling z:org.apache.spark.api.python.PythonRDD.collectAndServe.
[2024-10-02T18:57:02.046+0000] {spark_submit.py:488} INFO - : java.lang.IllegalStateException: SparkContext has been shutdown
[2024-10-02T18:57:02.046+0000] {spark_submit.py:488} INFO - at org.apache.spark.SparkContext.runJob(SparkContext.scala:2385)
[2024-10-02T18:57:02.046+0000] {spark_submit.py:488} INFO - at org.apache.spark.SparkContext.runJob(SparkContext.scala:2414)
[2024-10-02T18:57:02.046+0000] {spark_submit.py:488} INFO - at org.apache.spark.SparkContext.runJob(SparkContext.scala:2433)
[2024-10-02T18:57:02.046+0000] {spark_submit.py:488} INFO - at org.apache.spark.SparkContext.runJob(SparkContext.scala:2458)
[2024-10-02T18:57:02.047+0000] {spark_submit.py:488} INFO - at org.apache.spark.rdd.RDD.$anonfun$collect$1(RDD.scala:1049)
[2024-10-02T18:57:02.047+0000] {spark_submit.py:488} INFO - at org.apache.spark.rdd.RDDOperationScope$.withScope(RDDOperationScope.scala:151)
[2024-10-02T18:57:02.047+0000] {spark_submit.py:488} INFO - at org.apache.spark.rdd.RDDOperationScope$.withScope(RDDOperationScope.scala:112)
[2024-10-02T18:57:02.047+0000] {spark_submit.py:488} INFO - at org.apache.spark.rdd.RDD.withScope(RDD.scala:410)
[2024-10-02T18:57:02.048+0000] {spark_submit.py:488} INFO - at org.apache.spark.rdd.RDD.collect(RDD.scala:1048)
[2024-10-02T18:57:02.048+0000] {spark_submit.py:488} INFO - at org.apache.spark.api.python.PythonRDD$.collectAndServe(PythonRDD.scala:195)
[2024-10-02T18:57:02.048+0000] {spark_submit.py:488} INFO - at org.apache.spark.api.python.PythonRDD.collectAndServe(PythonRDD.scala)
[2024-10-02T18:57:02.049+0000] {spark_submit.py:488} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2024-10-02T18:57:02.050+0000] {spark_submit.py:488} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
[2024-10-02T18:57:02.051+0000] {spark_submit.py:488} INFO - at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2024-10-02T18:57:02.051+0000] {spark_submit.py:488} INFO - at java.base/java.lang.reflect.Method.invoke(Method.java:569)
[2024-10-02T18:57:02.052+0000] {spark_submit.py:488} INFO - at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
[2024-10-02T18:57:02.052+0000] {spark_submit.py:488} INFO - at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:374)
[2024-10-02T18:57:02.053+0000] {spark_submit.py:488} INFO - at py4j.Gateway.invoke(Gateway.java:282)
[2024-10-02T18:57:02.053+0000] {spark_submit.py:488} INFO - at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
[2024-10-02T18:57:02.054+0000] {spark_submit.py:488} INFO - at py4j.commands.CallCommand.execute(CallCommand.java:79)
[2024-10-02T18:57:02.054+0000] {spark_submit.py:488} INFO - at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2024-10-02T18:57:02.055+0000] {spark_submit.py:488} INFO - at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2024-10-02T18:57:02.056+0000] {spark_submit.py:488} INFO - at java.base/java.lang.Thread.run(Thread.java:840)
[2024-10-02T18:57:02.057+0000] {spark_submit.py:488} INFO - 
[2024-10-02T18:57:02.168+0000] {spark_submit.py:488} INFO - 24/10/02 18:57:02 INFO ShutdownHookManager: Shutdown hook called
[2024-10-02T18:57:02.170+0000] {spark_submit.py:488} INFO - 24/10/02 18:57:02 INFO ShutdownHookManager: Deleting directory /tmp/spark-3cd481a4-d46f-4610-a3d9-b8b3b205d5e2/pyspark-80329103-2540-4539-8b88-85f29e706676
[2024-10-02T18:57:02.175+0000] {spark_submit.py:488} INFO - 24/10/02 18:57:02 INFO ShutdownHookManager: Deleting directory /tmp/spark-37fbbf0c-3245-42f8-831b-254a0d7c6951
[2024-10-02T18:57:02.180+0000] {spark_submit.py:488} INFO - 24/10/02 18:57:02 INFO ShutdownHookManager: Deleting directory /tmp/spark-3cd481a4-d46f-4610-a3d9-b8b3b205d5e2
[2024-10-02T18:57:02.277+0000] {taskinstance.py:3310} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 157, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 419, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark-master:7077 --name arrow-spark jobs/wordcountjob.py. Error code is: 1.
[2024-10-02T18:57:02.286+0000] {taskinstance.py:1225} INFO - Marking task as FAILED. dag_id=sparking_flow, task_id=python_job, run_id=scheduled__2024-10-01T00:00:00+00:00, execution_date=20241001T000000, start_date=20241002T185548, end_date=20241002T185702
[2024-10-02T18:57:02.316+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-10-02T18:57:02.318+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 13455 for task python_job (Cannot execute: spark-submit --master spark://spark-master:7077 --name arrow-spark jobs/wordcountjob.py. Error code is: 1.; 78271)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 115, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3004, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3158, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3182, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 406, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 157, in execute
    self._hook.submit(self._application)
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 419, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master spark://spark-master:7077 --name arrow-spark jobs/wordcountjob.py. Error code is: 1.
[2024-10-02T18:57:02.353+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-10-02T18:57:02.401+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-02T18:57:02.409+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
